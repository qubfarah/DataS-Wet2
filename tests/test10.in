add_team 31397
add_jockey 11355 31397
add_jockey 2826 31397
add_team 4467
add_jockey 6641 4467
update_match 2826 6641
update_match 2826 6641
update_match 11355 6641
add_jockey 24690 4467
add_jockey 44787 31397
unite_by_record 3
add_jockey 3479 31397
add_team 35351
add_jockey 11660 35351
update_match 24690 11660
update_match 3479 11660
unite_by_record 2
add_team 39190
add_jockey 16723 39190
update_match 24690 16723
get_jockey_record 49654
update_match 44787 2826
get_jockey_record 11355
add_jockey 33839 39190
unite_by_record 1
add_jockey 44787 49604
get_jockey_record 11660
add_team 11627
add_jockey 14620 11627
update_match 33839 14620
add_jockey 23601 11627
add_jockey 11660 22188
get_jockey_record 49605
add_jockey 31299 11627
merge_teams 11627 31397
add_jockey 9938 31397
add_team 5929
add_team 14556
add_jockey 8058 5929
add_jockey 13693 31397
add_jockey 1835 36876
unite_by_record 29716
add_team 31397
update_match 31299 8058
update_match 16723 8058
get_jockey_record 23601
unite_by_record 0
add_jockey 41607 5929
unite_by_record 2
get_team_record 31397
get_jockey_record 6641
add_team 37802
add_jockey 40101 37802
update_match 16723 40101
get_jockey_record 3479
add_jockey 38145 31397
get_jockey_record 9938
get_team_record 31397
unite_by_record 1
add_jockey 25548 14556
get_jockey_record 38145
update_match 24690 25548
update_match 11660 25548
update_match 31299 25548
get_team_record 31397
add_jockey 37293 31397
update_match 38145 38145
add_jockey 16843 14556
get_team_record 36102
add_jockey 10677 14556
update_match 2826 10677
get_jockey_record 13693
get_jockey_record 43690
add_jockey 25857 31397
merge_teams 31397 14556
unite_by_record 0
add_jockey 17635 31397
add_team 49677
add_jockey 43874 49677
update_match 40101 43874
update_match 11660 43874
add_team 15709
update_match 15665 15665
add_jockey 35781 49677
update_match 43874 25548
add_jockey 8968 6274
add_jockey 31299 15709
update_match 11660 43874
update_match 33839 35781
update_match 17635 43874
unite_by_record 8421
unite_by_record 14595
add_jockey 33839 28165
update_match 24690 43874
add_team 49677
unite_by_record 0
add_jockey 23601 15709
update_match 38145 38145
unite_by_record 35167
add_team 43942
add_team 12635
add_jockey 42428 49677
update_match 37605 43874
get_jockey_record 11355
add_jockey 35311 10903
update_match 33839 35781
update_match 11660 43874
update_match 17635 43874
get_jockey_record 880
unite_by_record 8
add_jockey 24685 15709
update_match 25857 16843
add_jockey 46984 31397
update_match 35781 24685
add_jockey 18928 12635
add_jockey 14997 41411
unite_by_record 0
get_team_record 31397
update_match 3479 18928
unite_by_record 19771
update_match 40101 18928
update_match 42428 24685
get_jockey_record 6641
unite_by_record 4
update_match 46186 43984
get_team_record 15709
add_jockey 27569 31397
update_match 43874 18928
unite_by_record 43873
add_jockey 19886 12635
add_team 36439
unite_by_record 2
add_jockey 3956 31397
get_jockey_record 20183
add_jockey 8108 12635
merge_teams 12635 36439
update_match 17635 7961
add_jockey 13693 36439
add_jockey 39106 31397
unite_by_record 5
unite_by_record 3
update_match 11660 18928
merge_teams 15709 43942
unite_by_record 2
get_jockey_record 47473
get_team_record 11028
update_match 8058 18928
update_match 38145 8108
merge_teams 43942 31397
add_jockey 45210 5420
add_team 6911
add_jockey 44323 36439
add_jockey 20783 6911
update_match 40101 20783
get_team_record 17986
add_jockey 42428 31408
add_team 10184
unite_by_record 2579
add_jockey 33839 10184
get_jockey_record 40101
add_team 22314
get_team_record 31397
add_jockey 2621 6911
add_jockey 44214 36439
add_jockey 16406 10184
unite_by_record 16231
add_team 39183
get_team_record 10184
update_match 3479 44323
add_team 32718
merge_teams 31397 39183
update_match 19886 9938
add_jockey 17861 36439
add_team 47575
update_match 44214 6641
unite_by_record 0
add_jockey 40149 9229
add_jockey 4771 47575
update_match 3479 2621
update_match 44323 24685
update_match 39106 2621
add_jockey 45244 28054
add_team 46014
get_jockey_record 38145
update_match 27569 19886
add_jockey 9080 6911
add_team 1159
unite_by_record 0
add_team 32718
unite_by_record 5
update_match 8108 11355
add_team 1510
add_team 31097
add_jockey 2224 1159
add_jockey 9363 31397
unite_by_record 19601
add_team 1510
update_match 43874 44214
add_jockey 16723 43850
add_jockey 25250 31097
add_jockey 44787 47575
add_team 28182
get_team_record 22989
update_match 24685 2224
unite_by_record 3
update_match 2621 25548
unite_by_record 47155
add_team 30668
update_match 3479 44214
update_match 27569 2224
merge_teams 1159 36439
get_jockey_record 3479
add_jockey 23437 31097
add_jockey 3954 39237
add_jockey 44397 47575
add_jockey 16231 6911
get_jockey_record 46984
unite_by_record 0
update_match 14620 20783
update_match 31299 18928
add_jockey 2479 47575
update_match 38145 9080
add_jockey 41607 23768
get_jockey_record 13693
update_match 11660 25250
add_jockey 37820 10184
merge_teams 10184 22314
unite_by_record 0
update_match 37293 4771
unite_by_record 0
get_jockey_record 12972
unite_by_record 1
add_jockey 4632 31397
update_match 37293 44323
add_jockey 5865 46014
get_team_record 32847
update_match 8067 40101
update_match 25250 42428
update_match 41607 25250
add_jockey 41607 33113
add_team 6379
get_jockey_record 9080
update_match 9080 11660
merge_teams 10184 31397
get_jockey_record 46984
add_team 31097
unite_by_record 0
update_match 11355 16231
get_jockey_record 44787
update_match 17861 27890
add_jockey 2312 1510
add_team 6643
merge_teams 1510 31397
get_jockey_record 22191
update_match 11355 19886
add_jockey 3956 30668
get_team_record 32718
add_jockey 8108 31097
add_team 46483
update_match 7758 2764
unite_by_record 2155
add_jockey 7010 1159
