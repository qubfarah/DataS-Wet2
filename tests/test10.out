add_team: SUCCESS
add_jockey: SUCCESS
add_jockey: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
add_jockey: SUCCESS
add_jockey: SUCCESS
unite_by_record: SUCCESS
add_jockey: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
unite_by_record: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
get_jockey_record: FAILURE
update_match: FAILURE
get_jockey_record: SUCCESS, 1
add_jockey: SUCCESS
unite_by_record: SUCCESS
add_jockey: FAILURE
get_jockey_record: SUCCESS, -2
add_team: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
add_jockey: SUCCESS
add_jockey: FAILURE
get_jockey_record: FAILURE
add_jockey: SUCCESS
merge_teams: SUCCESS
add_jockey: SUCCESS
add_team: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
add_jockey: SUCCESS
add_jockey: FAILURE
unite_by_record: FAILURE
add_team: FAILURE
update_match: SUCCESS
update_match: SUCCESS
get_jockey_record: SUCCESS, 0
unite_by_record: INVALID_INPUT
add_jockey: SUCCESS
unite_by_record: SUCCESS
get_team_record: SUCCESS, 0
get_jockey_record: SUCCESS, -3
add_team: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
get_jockey_record: SUCCESS, 1
add_jockey: SUCCESS
get_jockey_record: SUCCESS, 0
get_team_record: SUCCESS, 1
unite_by_record: SUCCESS
add_jockey: SUCCESS
get_jockey_record: SUCCESS, 0
update_match: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
get_team_record: SUCCESS, 3
add_jockey: SUCCESS
update_match: INVALID_INPUT
add_jockey: SUCCESS
get_team_record: FAILURE
add_jockey: SUCCESS
update_match: SUCCESS
get_jockey_record: SUCCESS, 0
get_jockey_record: FAILURE
add_jockey: SUCCESS
merge_teams: SUCCESS
unite_by_record: INVALID_INPUT
add_jockey: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
add_team: SUCCESS
update_match: INVALID_INPUT
add_jockey: SUCCESS
update_match: SUCCESS
add_jockey: FAILURE
add_jockey: FAILURE
update_match: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
unite_by_record: FAILURE
unite_by_record: FAILURE
add_jockey: FAILURE
update_match: SUCCESS
add_team: FAILURE
unite_by_record: INVALID_INPUT
add_jockey: FAILURE
update_match: INVALID_INPUT
unite_by_record: FAILURE
add_team: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
update_match: FAILURE
get_jockey_record: SUCCESS, 1
add_jockey: FAILURE
update_match: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
get_jockey_record: FAILURE
unite_by_record: SUCCESS
add_jockey: SUCCESS
update_match: FAILURE
add_jockey: SUCCESS
update_match: SUCCESS
add_jockey: SUCCESS
add_jockey: FAILURE
unite_by_record: INVALID_INPUT
get_team_record: SUCCESS, 1
update_match: SUCCESS
unite_by_record: FAILURE
update_match: SUCCESS
update_match: SUCCESS
get_jockey_record: SUCCESS, -3
unite_by_record: FAILURE
update_match: FAILURE
get_team_record: SUCCESS, -2
add_jockey: SUCCESS
update_match: SUCCESS
unite_by_record: FAILURE
add_jockey: SUCCESS
add_team: SUCCESS
unite_by_record: FAILURE
add_jockey: SUCCESS
get_jockey_record: FAILURE
add_jockey: SUCCESS
merge_teams: SUCCESS
update_match: FAILURE
add_jockey: FAILURE
add_jockey: SUCCESS
unite_by_record: FAILURE
unite_by_record: FAILURE
update_match: SUCCESS
merge_teams: SUCCESS
unite_by_record: FAILURE
get_jockey_record: FAILURE
get_team_record: FAILURE
update_match: SUCCESS
update_match: SUCCESS
merge_teams: SUCCESS
add_jockey: FAILURE
add_team: SUCCESS
add_jockey: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
get_team_record: FAILURE
add_jockey: FAILURE
add_team: SUCCESS
unite_by_record: FAILURE
add_jockey: FAILURE
get_jockey_record: SUCCESS, 2
add_team: SUCCESS
get_team_record: SUCCESS, 7
add_jockey: SUCCESS
add_jockey: SUCCESS
add_jockey: SUCCESS
unite_by_record: FAILURE
add_team: SUCCESS
get_team_record: SUCCESS, 0
update_match: SUCCESS
add_team: SUCCESS
merge_teams: SUCCESS
update_match: SUCCESS
add_jockey: SUCCESS
add_team: SUCCESS
update_match: SUCCESS
unite_by_record: INVALID_INPUT
add_jockey: FAILURE
add_jockey: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
add_jockey: FAILURE
add_team: SUCCESS
get_jockey_record: SUCCESS, 1
update_match: SUCCESS
add_jockey: SUCCESS
add_team: SUCCESS
unite_by_record: INVALID_INPUT
add_team: FAILURE
unite_by_record: FAILURE
update_match: SUCCESS
add_team: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
add_jockey: SUCCESS
unite_by_record: FAILURE
add_team: FAILURE
update_match: SUCCESS
add_jockey: FAILURE
add_jockey: SUCCESS
add_jockey: FAILURE
add_team: SUCCESS
get_team_record: FAILURE
update_match: SUCCESS
unite_by_record: FAILURE
update_match: SUCCESS
unite_by_record: FAILURE
add_team: SUCCESS
update_match: SUCCESS
update_match: SUCCESS
merge_teams: SUCCESS
get_jockey_record: SUCCESS, 5
add_jockey: SUCCESS
add_jockey: FAILURE
add_jockey: SUCCESS
add_jockey: SUCCESS
get_jockey_record: SUCCESS, 0
unite_by_record: INVALID_INPUT
update_match: SUCCESS
update_match: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
add_jockey: FAILURE
get_jockey_record: SUCCESS, 0
update_match: SUCCESS
add_jockey: SUCCESS
merge_teams: SUCCESS
unite_by_record: INVALID_INPUT
update_match: SUCCESS
unite_by_record: INVALID_INPUT
get_jockey_record: FAILURE
unite_by_record: FAILURE
add_jockey: SUCCESS
update_match: SUCCESS
add_jockey: SUCCESS
get_team_record: FAILURE
update_match: FAILURE
update_match: SUCCESS
update_match: SUCCESS
add_jockey: FAILURE
add_team: SUCCESS
get_jockey_record: SUCCESS, -1
update_match: SUCCESS
merge_teams: SUCCESS
get_jockey_record: SUCCESS, 0
add_team: FAILURE
unite_by_record: INVALID_INPUT
update_match: SUCCESS
get_jockey_record: SUCCESS, 0
update_match: FAILURE
add_jockey: SUCCESS
add_team: SUCCESS
merge_teams: SUCCESS
get_jockey_record: FAILURE
update_match: SUCCESS
add_jockey: FAILURE
get_team_record: SUCCESS, 0
add_jockey: FAILURE
add_team: SUCCESS
update_match: FAILURE
unite_by_record: FAILURE
add_jockey: SUCCESS
